ddht/v5_1/abc.py

```diff
@@ -207,6 +228,10 @@ def initiate_session(
     def receive_session(self, remote_endpoint: Endpoint) -> SessionAPI:
         ...
 
+    @abstractmethod
+    def __contains__(self, session_id: uuid.UUID) -> bool:
```

Comment:
*nitpick*

Changing this class to inherit from `collections.abc.Container` will do this automatically.